src/App.jsx

```diff
@@ -14,70 +15,54 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export function App() {
+  const [selectedGood, setValue] = useState('Jam');
+  const isSelected = goodName => goodName === selectedGood;
+  const selectGood = goodName => setValue(isSelected(goodName) ? '' : goodName);
+  const isEmptyState = () => selectedGood === '';
+  const heading = isEmptyState()
+    ? `No goods selected`
+    : `${selectedGood} is selected`;
+  const selectedGoodClassName = 'has-background-success-light';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {heading}
+        {!isEmptyState() && (
+          <button
+            onClick={() => setValue('')}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        )}
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(goodName => (
+            <tr
```

Comment:
1. you should add a react key
2. you can use another syntax to make a variable inside rendering 
3. reuse `isActive` variable
```suggestion
          {goods.map(goodName => {
            const isActive = goodName === selectedGood;
            
            return (
              <tr
```